.github/workflows/ci-code.yml

```
@@ -27,6 +27,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ['3.9', '3.12']
+        database-backend: [psql, sqlite]
```

Instead of doubling the matrix size, you could run `sqlite` tests only for one of the Python versions, this should work

```suggestion
        database-backend: [psql]
        include:
          - python-version: '3.9'
            database-backend: 'sqlite'        
```

(using 3.9 would collect code coverage)